### PR TITLE
MAUI 829675 - UG for Undo Redo action on MAUI image editor.

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @Jeyasri-Murugan

 I have prepared content  for undo redo in Maui image editor control. Could you please review it and let me know if you have any concerns?

Regards,
Shalini.

### CODE REVIEW

File: MAUI/ImageEditor/undo-redo.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Undo Redo in .NET MAUI Image Editor Control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/undo-redo.md:3]: Undo Redo editing in .NET MAUI Image Editor Control | Syncfusion

File: MAUI/ImageEditor/undo-redo.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+description: Learn about Undo Redo support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Undo Redo in .NET MAUI Image Editor (SfImageEditor)
+
+One of the important features of the image editor control is to perform `Undo` and `Redo` operations for adding shapes, text, effects, and drawing paths.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/undo-redo.md:12]: One of the important features of the image editor control is to perform `Undo` and `Redo` to revert the changes.

File: MAUI/ImageEditor/undo-redo.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+title: Undo Redo editing in .NET MAUI Image Editor Control | Syncfusion
+description: Learn about Undo Redo support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Undo Redo in .NET MAUI Image Editor (SfImageEditor)
+
+One of the important features of the image editor control is to perform `Undo` and `Redo` to revert the edited changes.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/undo-redo.md]: Changed a content

File: MAUI/ImageEditor/reset.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ documentation: ug
 
 # Reset image using the .NET MAUI Image Editor (SfImageEditor)
 
+The Image Editor control in .NET MAUI allows you to reset the edited image to the original loaded image.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/reset.md]: Modified a content

File: MAUI/ImageEditor/undo-redo.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/undo-redo.md]: Changed a title

File: MAUI/ImageEditor/undo-redo.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/undo-redo.md]: Modified a content

**Review by Jeyasri-Murugan — APPROVED** [review]
